src/users/aliases.md

```diff
@@ -0,0 +1,27 @@
+# Aliases
+
+Aliases are individual emojis that you can name and then be used through that name.
+Aliases are user dependent, meaning that only the user that named or imported one can use it through that name. 
+For more information regarding how to name server wide aliases refer to [adding emojis through packs and aliases](../guild/Through_packs_and_aliases.md).
+
+You can find the full list of Aliases you have through the use of the command `/alias mine` or `/alias dm`(same response but sent to dm)
+
+You can search for emojis using the following command `/alias search termsOfYourSearch`. 
+The search functionality also supports the following filters:
+- `exact`: Shows only the emojis with names that exactly match your search terms.
+- `animated`: If set to `True` only show animated emojis. If `False` only static ones.
```

Comment:
Would be better with 'leave empty to show all emotes' or something to that effect